miqcli/cli/main.py

```diff
@@ -272,8 +205,26 @@ def get_command(self, ctx, name):
         return cmd
 
 
+# set context settings w/ our configuration
+config = Config({})
+
+# check lowest precedence /etc first
+if check_yaml(MIQCLI_CFG_FILE_LOC)["exists"]:
```

Comment:
you don't need the check_yaml file anymore. from_yaml function does the work for you. If you set silent=True nothing happens if the file doesn't exist or if the file is invalid. Maybe silent should be set to false? Either way, you don't need check_yaml anymore. That can be removed from the source code.